api/src/main/java/org/openmrs/Encounter.java

```diff
@@ -488,8 +524,18 @@ public void addCondition(Condition condition) {
 	 * @param condition - the condition to remove
 	 */
 	public void removeCondition(Condition condition) {
```

Comment:
@icrc-loliveira i think you can kindly add a test coverage for removeCondition to make sure it voids the condition and remove it , you can a sample look of it here as its done for the[ encounter providers](https://github.com/openmrs/openmrs-core/blob/2f3e2928f84498c431a775a86930956a87a15377/api/src/test/java/org/openmrs/EncounterTest.java#L1247)